### PR TITLE
add link to bindings doc in the getting started local dev section

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

File: pages/cloudflare/get-started.mdx [hunk 1]
```diff
@@ -125,6 +125,8 @@ You can continue to run `next dev` when developing locally.
 
 In step 3, we also added the `npm run preview:worker`, which allows you to quickly preview your app running locally in the Workers runtime, rather than in Node.js. This allows you to test changes in the same runtime as your app will run in when deployed to Cloudflare.
 
+During local development, either via `next dev` or `npm run preview:worker`, you can access local versions of Cloudflare bindings as indicated in the [bindings documentation](./bindings).
```
Comment (IgorMinar) [inline pages/cloudflare/get-started.mdx:128]: I'd move this one paragraph above so that the first sentence and this paragraph are together.

Comment (IgorMinar) [reply pages/cloudflare/get-started.mdx:128]: the rest looks good.

**Review by IgorMinar — APPROVED** [review]

File: pages/cloudflare/get-started.mdx (hunk 1 above)
Comment (dario-piotrowicz) [reply pages/cloudflare/get-started.mdx]: I've moved it and removed the sentence, please let me know if this looks good to you 🙂

**Review by IgorMinar — APPROVED** [review]
lgtm

**Pull request merged**